less/forum/Post.less

```diff
@@ -116,6 +116,14 @@
     border-bottom: 2px dotted @body-bg;
     margin: 1em 0;
   }
+  code {
+    font-family: source-code-pro, Monaco, Consolas, "Courier New", monospace;
+    padding: 2px 6px;
+    background: #60646D;
+    color: #256FC7;
```

Comment:
@franzliedke What variable should I use, then?
I just had a look at them, but I'm not good with color codes (RGB or HEX) at all :wink:.